azavea/raster-foundry
# Add and use nodeSelector component in split map view for lab

## Overview
Add and use nodeSelector component in split map view for lab
Misc fixes:
- Update fonts
- Fix outline on node dropdown menu
- Comment out "Add Note" button
- Disable "Apply Changes" when request is in flight
- Optimistically remove warnings when making requests
- Add some padding to the map split slider
- Fix number input stepping for input and constant nodes
- Refactor some diagramContainer logic into dagUtils service


### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Demo
![image](https://user-images.githubusercontent.com/4392704/28432826-6ec99048-6d57-11e7-8da3-6aaad9f4aab7.png)
![image](https://user-images.githubusercontent.com/4392704/28432976-0faa7658-6d58-11e7-94a8-0d54a6ca9bba.png)


### Notes

## Testing Instructions

* Verify the miscellaneous fixes listed above
* Verify that selecting a node from the split view node label works
* Verify that closing the split view by the map toggle button as well as the label close button works
* Verify that text search on the nodes by label works

Closes #2286 
Closes #2164 
Closes #2167 
Closes #2165 
Closes #2166
